src/Workspaces/Core/Portable/CodeGeneration/AbstractCodeGenerationService_FindDeclaration.cs

```diff
@@ -108,6 +108,15 @@ public bool CanAddTo(SyntaxNode destination, Solution solution, CancellationToke
             return availableIndices != null && availableIndices.Any(b => b);
         }
 
+        public async Task<SyntaxNode> FindMostRelevantNameSpaceOrTypeDeclarationAsync(
+            Solution solution,
+            INamespaceOrTypeSymbol namespaceOrType,
+            CodeGenerationOptions options,
+            CancellationToken cancellationToken)
+        {
+            return (await FindMostRelevantDeclarationAsync(solution, namespaceOrType, options ?? CodeGenerationOptions.Default, cancellationToken)).declaration;
```

Comment:
please don't mix-match 'awaits in expressions like this.  first, the expression is far too complex to even see on github. second, ti's just confusing to read.  Feel free to just make a local, then return the property off the local.